docs/source/tune_cli.rst

```diff
@@ -207,5 +209,5 @@ The ``tune validate <config>`` command will validate that your config is formatt
 
 .. code-block:: bash
 
-    $ tune validate recipes/configs/full_finetune_distributed.yaml
+    $ tune validate recipes/configs/llama3/8B_full.yaml
```

Comment:
Can we modify this a bit more and make the path look like a custom config path, e.g. `my_configs/llama3/8B_full.yaml`?